WordPress/src/main/java/org/wordpress/android/util/PerAppLocaleManager.kt

```diff
@@ -155,4 +95,30 @@ class PerAppLocaleManager @Inject constructor(
         // update Reader tags as they need be localized
         ReaderUpdateServiceStarter.startService(getContext(), EnumSet.of(UpdateTask.TAGS))
     }
+
+    /**
+     * This routine can be helpful during development to reset the app locale
+     */
+    @Suppress("unused")
+    fun resetApplicationLocale() {
+        AppCompatDelegate.setApplicationLocales(LocaleListCompat.getEmptyLocaleList())
+    }
+
+    companion object {
```

Comment:
Question (❓): Why creating the companion object now? 🤔 
Suggestion (💡): Although I kind of understand the need for doing that, I recommend reverting to using those function non-statically, like it was done before, especially after the `LocalManager.getLanguage()` is replaced by `PerAppLocaleManager.getCurrentLocaleLanguageCode()` (which I understand is the only reason this happened in the first place, right? 🤔 

FYI: Btw, I am only questioning and suggesting this because doing that makes this whole solution more (unit) testable.

Reply:
You are correct that making this static was so `LocalManager.getLanguage()` could access it. I do plan to make it non-static after `LocalManager.getLanguage()` is dropped, but I'd prefer to make that change in a separate PR.

Reply:
Awesome, works for me Nick! 👍